### ISSUE TITLE
Notification read/unread state should be indicated to screen reader in notification drawer

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#5889]
**Describe the issue. What is the expected and unexpected behavior?**
When read/unread state changes in notification drawer, there should be an indication to screen readers what has changed. (Sighted users can see that the styling changes, but there is no indication to screen reader users.) We could likely provide a label or screen reader text to add this context.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**
Test notification drawer with VO and notice that when notification is read, there is no new information given to screen readers.

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Bug

**Comment by nicolethoen** [comment]
This will likely be fixed using something similar to this PR: https://github.com/patternfly/patternfly-react/pull/6664

Screen reader text should be fully configurable, but also have a default value set. It should also be wrapped using the `.pf-u-screen-reader` class so that it is only visible to screen readers.

This is a great opportunity to learn how to use a screen reader to test the update!

**Comment by tpakkal1** [comment]
I can be assigned this!
